__tests__/test.js

```diff
@@ -92,6 +92,44 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH request", () => {
+    test("should accept an object in the form { inc_votes: newVote }, returning 200 and the updated article", () => {
+      const expectedArticle = {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 110,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
```

Comment:
I'd consider adding an object representing the data that is being sent in the PATCH request, for example:

**const expectedArticle = {inc_votes: 10}**

Rather than the whole article, this keeps it concise and you can check the correct amount of votes comes back 😄